torch/_prims/context.py

```diff
@@ -175,15 +175,20 @@ def __torch_function__(
         # implementations.
         # There're other ways to implement this functionality,
         # see https://github.com/pytorch/pytorch/pull/82657#discussion_r939776417
+        decomp = False
```

Comment:
nitpick: debug code should be removed.

Reply:
Thanks for the catch. I forgot to remove that line.